### PR TITLE
Performance measurements screens

### PR DESCRIPTION
**Pull request by sarathms**
For NDT and Dash measurements.

### CODE REVIEW

**Comment by sarathms** [comment]
@bassosimone I have merged `master` into this branch. You can review the relevant changes now.

File: renderer/components/nettests/index.js [hunk 1]
```diff
@@ -50,8 +52,8 @@ export const tests = {
   facebook_messenger,
   telegram,
   whatsapp,
-  ndt: web_connectivity,
-  dash: web_connectivity,
+  ndt,
+  dash,
```
Comment (bassosimone) [inline renderer/components/nettests/index.js]: I believe you have a bunch of TODOs comments to zap, right?

File: renderer/components/nettests/performance/Dash.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import formatSpeed from '../../formatSpeed'
+import performanceTestGroup from './index'
+
+const Dash = ({measurement, render}) => {
+  const testKeys = JSON.parse(measurement.test_keys)
+  const {
+    connect_latency,
+    median_bitrate,
+    min_playout_delay
+  } = testKeys
```
Comment (bassosimone) [inline renderer/components/nettests/performance/Dash.js]: what is this doing?

File: renderer/components/nettests/performance/Dash.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            value={min_playout_delay}
+          />
+        </Box>
+      </Flex>
+    </Box>
+  )
+
+  return (
+    <div>
+      {render({
```
Comment (bassosimone) [inline renderer/components/nettests/performance/Dash.js]: why is `render` here? I seem to not understand where it is needed

File: renderer/components/nettests/performance/Dash.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+
+Dash.propTypes = {
+  measurement: PropTypes.object,
+  render: PropTypes.func
+}
+
+export { Dash }
+
+// Metadata for the nettest
```
Comment (bassosimone) [inline renderer/components/nettests/performance/Dash.js]: 👍 for consistency!

File: renderer/components/nettests/performance/NDT.js [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+import performanceTestGroup from './index'
+
+const NDT = ({measurement, render}) => {
+  const testKeys = JSON.parse(measurement.test_keys)
+  const {
+    download,
+    upload,
+    ping,
+    server = 'Not in test_keys',
+    packet_loss,
```
Comment (bassosimone) [inline renderer/components/nettests/performance/NDT.js]: Gone

File: renderer/components/nettests/performance/NDT.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    out_of_order,
```
Comment (bassosimone) [inline renderer/components/nettests/performance/NDT.js]: Gone

File: renderer/components/nettests/performance/NDT.js [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+    avg_rtt,
+    max_rtt,
```
Comment (bassosimone) [inline renderer/components/nettests/performance/NDT.js]: Dangerous to rely on: no easy way to reliably compute it

File: renderer/components/nettests/performance/NDT.js [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+    mss,
+    timeouts
```
Comment (bassosimone) [inline renderer/components/nettests/performance/NDT.js]: Gone

File: renderer/components/nettests/performance/NDT.js [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+    </Box>
+  )
+
+  return (
+    <div>
+      {render({
+        hero: NDTHero,
+        heroBG: performanceTestGroup.color,
+        details: <NDTetails />
+
```
Comment (bassosimone) [inline renderer/components/nettests/performance/NDT.js]: extra blank line?

File: renderer/components/nettests/performance/Dash.js (hunk 2 above)
Comment (sarathms) [reply renderer/components/nettests/performance/Dash.js]: It is called [object destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring). Equivalent to:
```
const connect_latency = testKeys.connect_latency
...
```

File: renderer/components/nettests/performance/Dash.js (hunk 3 above)
Comment (sarathms) [reply renderer/components/nettests/performance/Dash.js]: This is the pattern I was talking about in [the WebConnectivity PR](https://github.com/ooni/probe-desktop/pull/49#issuecomment-539690345). We receive `render` function from the parent component which constructs what is to be rendered here based on `details` from the child here.

File: renderer/components/nettests/performance/Dash.js (hunk 2 above)
Comment (bassosimone) [reply renderer/components/nettests/performance/Dash.js]: oh, wow!

File: renderer/components/nettests/performance/Dash.js (hunk 3 above)
Comment (bassosimone) [reply renderer/components/nettests/performance/Dash.js]: ah, gotcha!

File: renderer/components/nettests/index.js (hunk 1 above)
Comment (sarathms) [reply renderer/components/nettests/index.js]: Zapped :zap:

File: renderer/components/nettests/performance/NDT.js (hunk 9 above)
Comment (sarathms) [reply renderer/components/nettests/performance/NDT.js]: removed

**Pull request merged**